File: src/main/java/application/service/StudentService.java
package application.service;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import application.model.Student;
import application.persistence.StudentRepository;

@Service
public class StudentService{
	
	@Autowired
	private StudentRepository studRepo;
	
	
	public Student getStudentByFileNumber(int fileNumber) {
		return studRepo.findByStudentFileNumber(fileNumber);
	}

	public void save(Student s) {
		studRepo.save(s);
	}

}

File: src/main/kotlin/ar/edu/unq/eiroalando/service/StudentService.kt
package ar.edu.unq.eiroalando.service

import org.springframework.stereotype.Service
import ar.edu.unq.eiroalando.model.Student
import ar.edu.unq.eiroalando.persistence.StudentRepository
import org.springframework.beans.factory.annotation.Autowired



@Service
class StudentService {

    @Autowired
    lateinit var studRepo: StudentRepository


    fun getStudentByFileNumber(fileNumber: Int): Student {
        return studRepo!!.findByStudentFileNumber(fileNumber)
    }

    fun save(s: Student) {
        studRepo!!.save(s)
    }
}